opencivicdata/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
ideally this would only be present in Python 2.7 version, as this won't be strictly backwards-compatible

though I'm amenable to this potentially being a breaking change if we can test that it works fine to upgrade to/from versions without this